src/bin/miri.rs

```diff
@@ -40,9 +36,12 @@ impl<'a> CompilerCalls<'a> for MiriCompilerCalls {
 
             let tcx = state.tcx.unwrap();
             let mir_map = state.mir_map.unwrap();
-            let (span, mir, def_id) = get_main(tcx, mir_map);
+
+            let (node_id, span) = state.session.entry_fn.borrow().expect("no main or start function found");
             println!("found `main` function at: {:?}", span);
```

Comment:
Still a `println` in the overall diff https://github.com/solson/miri/pull/23/files#diff-c3d602c5c8035a16699ce9c015bfeceaR41